userManagement/templates/userManagement/userAccount.html

```diff
@@ -0,0 +1,83 @@
+{% extends "itemManagement/chrome_template.html" %}
+{% load crispy_forms_tags %}
+{% block body %}
+    <h1>Staff</h1>
+    <div class="table-responsive-sm">
+        <table id="staffTable" class="table table-hover ">
+          <thead>
+            <tr>
+              <th scope="col">Name</th>
+              <th scope="col">Email</th>
+              <th scope="col">Role</th>
+              <th scope="col"></th>
+            </tr>
+          </thead>
+          <tbody>
+          {% for person in staff %}
+              {% if person.deleteFlag == False %}
+                <tr>
+                    <th>{{ person.name }}</th>
+                    <td>{{ person.email }}</td>
+                <td>
+                    {% if person.email !=  request.user.email %}
+                        <div class="dropdown">
+                            <button class="btn btn-sm btn-outline-secondary dropdown-toggle" type="button" id="dropdownMenuButton" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+                                {% if person.is_superuser %}
+                                    Admin
+                                {% else %}
+                                    Lab Assistance
```

Comment:
Lab Assistance, not assistance.

Reply:
Sorry, which line is it?

Reply:
It can be easier to  see everything in the "Files Changed" tab